README.md

```diff
@@ -47,6 +48,8 @@ new globalizePlugin({
 
 *output* is the name scheme of the built files.
 
+*transformExclusions* an array of functions to test on filepaths, and optionally exclude matching files from the dependency transformations. See [react-globalize-webpack-plugin](https://github.com/rxaviers/react-globalize-webpack-plugin) for an example usage.
+
```

Comment:
Follow up comment in https://github.com/rxaviers/globalize-webpack-plugin/pull/9#issuecomment-157363068